styles/components/cards.css

```diff
@@ -0,0 +1,156 @@
+.card {
+  display: flex;
+  border: 1px solid var(--inactive-color);
+  flex-direction: column;
+  margin: 1rem 0.5rem;
+  max-height: 30%;
+  max-width: 22.5rem;
+  min-width: 12.5rem;
+  border-radius: var(--radius);
+  font-family: Roboto, sans-serif;
+}
+
+.card-ecom {
+  width: 13.125rem;
+}
+
+.card-body {
+  display: flex;
+  flex-direction: column;
+}
+
+.card-heading {
+  padding: 1rem;
+}
+
+/* To show heading at top before img */
+.card-heading-top .card-heading {
+  order: -1;
+}
+
+.card-title {
+  font-size: 1.25rem;
```

Comment:
Suggestion: prefer using classes or variables for font-size, font-weight. This will ensure consistency throughout the project. You may include it in a typography stylesheet.

Reply:
Thanks Tanay..will include it in my typography component

Reply:
Thanks Tanay..will include it in my typography component